rtos/Thread.h

```diff
@@ -69,11 +69,12 @@ class Thread {
       @param   priority       initial priority of the thread function. (default: osPriorityNormal).
       @param   stack_size     stack size (in bytes) requirements for the thread function. (default: OS_STACK_SIZE).
       @param   stack_mem      pointer to the stack area to be used by this thread (default: NULL).
+      @param   name           name to be used for this thread (default: NULL)
```

Comment:
Maybe should mention that user must keep name allocate during the lifetime of the thread?